src/components/Header/styles.js

```diff
@@ -0,0 +1,56 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import { css } from "styled-components";
+
+const white = css`
+var(--white);
+`
+
+export const Container = styled.nav`
```

Comment:
I see u using this Container styled component a lot along with some components above. I understand it is a nav element but I think you should rename it for code readability purposes... Maybe NavContainer?

Reply:
And if you don't mind please use commit messages in English and present tense verbs... for example "refactor code to styled-components" or "correct responsiveness" but if language is a barrier for you no problem do the way you are used to

Reply:
Okay, I will follow this tips on my next PR. I agree with you on the question about name of the style components.

Reply:
Thanks.... I am still reviewing your PR. So far I like what I see. 🤲🏻